routes/api/user/update-user.js

```diff
@@ -7,5 +7,50 @@ var router = express.Router();
 
 console.log('update-user');
 
+router.get("/api/users/:id", function(req, res) {
```

Comment:
`router.get` should be `router.put`

Let's change `/api/users/:id` to `/api/users` since the `where` clause is pulling the `id` from the `req.body` and not the `req.params`